pkg/harness/stackrun/v1/types.go

```diff
@@ -43,6 +46,13 @@ func (in *StackRun) MaxSeverity() int {
 }
 
 func (in *StackRun) FromStackRunBaseFragment(fragment *gqlclient.StackRunBaseFragment) *StackRun {
+	var parallelism *int64
```

Comment:
not the end of the world, but this would be more straightforward to do as:

```elixir
stack := &StackRun{...}
if tf := fragment.Configuration.Terraform; tf != nil {
  stack.Parallelism = tf.Parallelism
  refresh = tf.Refresh
}
```

Also is `Configuration` not a pointer?  If it can be then it can still cause a segfault